examples/train_softmax.py

```diff
@@ -0,0 +1,149 @@
+import torch
+import argparse
+import os
+import pickle
+import datetime
+from torch import nn
+from binding_prediction.models import BindingModel
+from binding_prediction.dataset import ComparisonDrugProteinDataset, collate_fn_triplet
+from binding_prediction import pretrained_language_models
+from examples.train_nce import get_targets, run_model_on_batch, initialize_logging
```

Comment:
I'm getting an error here when trying to run this code.  This seems to be resolved when I replace this with
```suggestion
from train_nce import get_targets, run_model_on_batch, initialize_logging
```